epub33/reports/epub-properties-use.md

```diff
@@ -0,0 +1,604 @@
+# EPUB 3.3 Metadata Usage Report
+
+## Candidate Recommendation Exit Criteria
+
+The EPUB Working Group intends to exit the Candidate Recommendation stage and submit
+the EPUB 3.3 specification for consideration as a W3C Proposed Recommendation
+after documenting implementation of each feature.
+
+For this specification to advance to Proposed Recommendation, it has to be
+proven that metadata defined and required in this specification have sufficient usage by the
+target communities. This metadata falls into two categories:
+
+1. metadata for expressing information about the publication in the package document; and
+2. metadata for expressing preferred rendering of the content
+
+Usage of these properties means that they are regularly included in the Package Document
+metadata for their EPUB Publications (as appropriate for each title).
+
+## Publisher Implementations
+
+### Meta Properties Vocabulary
+
+The following table lists publishers who have stated that they are currently using
+the meta properties in production.
+
```

Comment:
This is a general comment, both for this section and the other document: it may be better to put active links to the specification themselves. It helps the reviewers to find their way through the report.

Also, we should discuss whether to set up the two-way links between the specifications and this document the same way as we have done for the functional tests (recognizing that, if we do the latter, it may be better to convert these documents into HTML, which is of course doable but it is a drag). Again, this may help the reviewers to see the completeness (or not...) of our "tests".

Cc @dlazin

Reply:
> we should discuss whether to set up the two-way links

It probably wouldn't be too hard to do this. I assume the script for the test links checks if specStatus=ED before generating?

Reply:
This is done by respec, not by my script; my stuff just generates the reports. 

However. I realize now that it straight out of the box to use the respec trick. Respec uses the `testSuiteURI`: it sees a `data-test` value (e.g., `#pgk=lang_pkg`), concatenates that with the value of `testSuiteURI` and then creates the small pull-down values (and yes, it does it only for the editors' draft). I do not think it is prepared to use several `testSuiteURI` values.

That being said, there is already a `data-test-display.js` script that massages those boxes a bit. It would be possible to find out, somehow, that those links are to be used with a different URI...

But we still have to have the right target fragment. Adding a fragment to a markdown my be a pain.

Reply:
I've linked all the properties to their definitions in the latest commit. I've also added links to any other references (not just to the specs but also tools).

For completeness, I've also added a paragraph to each section saying where the properties are used. Might help with understanding what the properties do. Worst case it doesn't harm anything.

Reply:
I haven't tried to link from the specification to the report.